ReactNativeClient/lib/models/Folder.js

```diff
@@ -190,6 +190,29 @@ class Folder extends BaseItem {
 		return output;
 	}
 
+	static async sortLocale(folders, orderDir = 'ASC') {
+
+		orderDir = orderDir.toUpperCase();
```

Comment:
No need to make it upperCase. The order should be provided in the right format.

Reply:
@Rishgod, you make the conversation as resolved, but you don't make the change or even discuss it?

Reply:
Sorry sir, I thought I did make the change, my bad, I'll refrain from resolving conversations in the future and let the reviewer do it, I can see it becoming a nuisance for the review my bad!